buildpack.md

```diff
@@ -197,6 +195,21 @@ Executable: `<layers>/<layer>/exec.d/<process>/<executable>`, Working Dir: `<app
 | Standard error     | Logs (warnings, errors)
 | [†](README.md#linux-only)FD 3 or [‡](README.md#windows-only)`<handle>` | Launch time environment variables (see [Exec.d Output](#execd-output-toml))
 
+### Process Definition
+
+Each buildpack may write a [launch.toml](#launchtoml-toml) file in order to define processes that the launcher can run on the app image.
+
+| Field         | Type            | Definition                                                                                   |
+|---------------|-----------------|----------------------------------------------------------------------------------------------|
+| `type`        | String          | An identifier for the process                                                          |
+| `command`     | Array of string | The command to execute, followed by arguments that should always be provided [^command-args] |
+| `args`        | Array of string | Default arguments to the command that can be overridden by the user                          |
+| `default`     | Boolean         | If `true`, use this as the default process for the app image                                 |
+| `working-dir` | Path            | Working directory for this process in the app image                                          |
```

Comment:
This information is somewhat duplicated in the launch.toml section below. I wonder if we could consolidate it in one place.

Reply:
Consolidated in launch.toml section in de2636c